change/@fluentui-babel-make-styles-9be0a866-541e-411f-9491-255721765189.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "adopt API changes from makeStyles",
```

not sure how useful this changelog entry is. don't we want to have something like
```suggestion
  "comment": "makeStyles: remove \"unstable_cssPriority\"",
```